src/shim/preload_syscall.c

```diff
@@ -0,0 +1,196 @@
+#include <assert.h>
+#include <dlfcn.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <poll.h>
+#include <stdarg.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <sys/epoll.h>
```

Comment:
We could probably drop a lot of these now, but I wouldn't spend too much time figuring out exactly which ones. Eventually we could rerun the iwyu tool again.